interbtc-spec/docs/source/spec/oracle.rst

```diff
@@ -6,77 +6,57 @@ Exchange Rate Oracle
 
 
 Data Model
 ~~~~~~~~~~
 
-Constants
----------
-
-GRANULARITY
-...........
-
-The granularity of the exchange rate. The granularity is set to :math:`10^{-5}`.
-
-
 Scalars
 -------
 
-ExchangeRateBtcInDot
-....................
-
-The BTC in DOT exchange rate. This exchange rate is used to determine how much collateral is required to issue a specific amount of interbtc. 
-
-.. note:: If the ``ExchangeRate`` is set to 1238763, it translates to :math:`12.38763` as the last five digits are used for the floating point (as defined by the ``GRANULARITY``).
-
-
-SatoshiPerBytesFast
-...................
-
-The estimated Satoshis per bytes required to get a Bitcoin transaction included in the next block.
-
-
-SatoshiPerBytesMedium
-.....................
+ExchangeRate
+............
 
-The estimated Satoshis per bytes required to get a Bitcoin transaction included in the next three blocks (about 30 min).
+The base exchange rate MUST be stored in the smallest denomination of the currency pair (e.g., Planck per Satoshi). This exchange rate is used to determine how much collateral is required to issue a specific amount of interBTC.
 
+.. note:: If the exchange rate between BTC and DOT is 2308 (i.e. 1 BTC = 2308 DOT) then we can convert to the base rate as follows:
+    ``planck_per_satoshi = dot_per_btc * (10**dot_decimals / 10**btc_decimals)``
+    ``230800 = 2308 * (10**10 / 10**8)``
 
-SatoshiPerBytesSlow
-...................
+The exchange rate MUST be stored in a 128-bit unsigned fixed-point representation.
 
-The estimated Satoshis per bytes required to get a Bitcoin transaction included in the six blocks (about 1 hour).
+SatoshiPerBytes
+...............
 
+The estimated Satoshis per bytes required to get a Bitcoin transaction included - see the :ref:`btcTxFeesPerByte`.
 
 MaxDelay
 ........
 
 The maximum delay in seconds between incoming calls providing exchange rate data. If the Exchange Rate Oracle receives no data for more than this period, the BTC Parachain enters an ``Error`` state with a ``ORACLE_OFFLINE`` error cause.
 
-
 LastExchangeRateTime
 ....................
 
 UNIX timestamp indicating when the last exchange rate data was received. 
 
 
-Enums
------
-
-InclusionEstimate
-.................
+Structs
+-------
 
-The estimated time until when a BTC transaction is included based on the Satoshi per byte fee.
+.. _btcTxFeesPerByte:
 
-* ``FAST: 0`` - the fee to include a BTC transaction within the next block.
+BtcTxFeesPerByte
+................
 
-* ``MEDIUM: 1``- the fee to include a BTC transaction within the next three blocks (~30 min)).
+The estimated inclusion time for a Bitcoin transaction MUST be stored in Satoshis per byte.
 
-* ``SLOW: 2`` - the fee to include a BTC transaction within the six blocks  (~60 min).
+* ``fast`` - the fee to include a BTC transaction within the next block.
```

Comment:
Typically we display it in a table and include types. But I am fine with merging the PR now